packages/web/components/control/token-select-with-drawer.tsx

```diff
@@ -9,13 +9,13 @@ import { TokenSelectDrawer } from "~/components/drawers/token-select-drawer";
 import { Disableable } from "~/components/types";
 import { EventName, SwapPage } from "~/config";
 import { useAmplitudeAnalytics, useWindowSize } from "~/hooks";
-import { SwapState } from "~/hooks/use-swap";
+import { SwapAssets } from "~/hooks/use-swap";
 
 /** Will display balances if provided `CoinPretty` objects. Assumes denoms are unique. */
 export const TokenSelectWithDrawer: FunctionComponent<
   {
     isFromSelect: boolean;
-    swapState: SwapState;
```

Comment:
For example, this component would rerender on quote update when it only should rerender if we change to and from assets